deps/wazuh_testing/wazuh_testing/process_resource_monitoring/src/process_resource_monitoring/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Process resource usage monitoring tool.
+
+This package contains the following modules:
+
+monitor -- Process and child processes monitoring using one process per
+           instance (one thread). Tracks several resource usage metrics
+
+disk_usage_tracker -- Track disk usage of files/directories over time. Show 
```

Comment:
```suggestion
disk_usage_tracker -- Track disk usage of files/directories over time. Show
```

Remove trailing whitespace.

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5747/commits/42bdbf6a440e7627707ed2e0df6b151f843ec8f1